README.md

```diff
@@ -15,18 +15,23 @@ After installation, the `proxygen` executable is available. Typing `proxygen` di
 
 The CLI requires client credentials, which can be obtained from the `platforms-api-producer-support` Slack channel.
 
-All users should also have individual credentials. `proxygen-cli` needs to know about them.
-You can setup the credentials in `proxygen-cli` using:
+The Proxygen CLI client credentials are setup now as part of proxygen cli package.
+
+Now as part of credential setting only input values for prompts  `username`, and `password` 
+
+proxygen credentials set
 ```
+
+if in case client credentials are not setup as part of package installation or you want to update  the credentials in the future, use: 
+
 proxygen credentials set
 ```
-This command prompts you to enter your `client_id`, `client_secret`, `username`, and `password`. 
-To update credentials in the future, use:
+This command prompts you to enter your `client_id`, `client_secret`, `username`, and `password`.
+and also this will help to update the default client credentials (`client_id`, `client_secret`) in the future, use:
 ```
```

Comment:
This is good for your current ticket but it seems the README isn't clear on the difference between user access and client access. Maybe the package isn't quite configured right for the 2 as well. Maybe we can have a think about this and create a follow on ticket if we're not happy as it would be out of scope

Reply:
Sure Alex Will go through it and I think we need Follow up ticket on this.